frontend/components/text-classifier/header/TextClassificationHeader.vue

```diff
@@ -102,7 +102,7 @@ export default {
         const labelsToSend = this.labelsFactoryBySingleOrMultiLabel(
           labels,
           labelsToRemove,
-          record.currentAnnotation?.labels
+          record.currentAnnotation?.labels || []
```

Comment:
1/ Clean code : don't do too much thing in one line : 
 It's better to do the test before in a specific variable and pass this variable as argument

2/ it's a very little comment, but maybe you want to use nullish coallescing instead of || => record.currentAnnotation?.labels ?? []